src/tests/unit/tests/DetailsView/components/target-change-dialog.test.tsx

```diff
@@ -26,7 +25,7 @@ describe('TargetChangeDialog test set for prev tab null', () => {
         };
 
         urlParserMock
-            .setup(urlParserObject => urlParserObject.areURLHostNamesEqual(prevTab, newTab.url))
+            .setup(urlParserObject => urlParserObject.areURLHostNamesEqual(It.isAny(), newTab.url))
```

Comment:
why do you need to make this change?

Reply:
Because the `urlParser` accepts a `string` and since this code checks that the mocked is never called (see `Times.never()`). This change actually setups the test correctly, since now it represents that whatever the value the mock is never called.

Reply:
Thanks, it make total sense when verifying a mock is never called.